bears/general/AnnotationBear.py

```diff
@@ -7,12 +7,17 @@
 from coala_utils.string_processing.Core import unescaped_search_for
 
 
+@Language
+class UnknownLanguage:
+    pass
```

Comment:
I assume you mean in coala.  Do you also want it appearing in `coalib/bearlib/languages/__init__.py` /  `LanguageUberMeta.all` ?

If we include it here, at least temporarily, we don't need to do a release of `coala` to fix this bug.  It could just be part of coala-bears 0.9.2 .